(dp1
S'output'
p2
S"<type 'exceptions.UnboundLocalError'> local variable 'form' referenced before assignment"
p3
sS'layer'
p4
S'/Users/rafihaque/Documents/cmps183/web2py/applications/hicappo/controllers/default.py'
p5
sS'code'
p6
S'<code object <module> at 0x102ebe5b0, file "/Users/rafihaque/Documents/cmps183/web2py/applications/hicappo/controllers/default.py", line 12>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/Users/rafihaque/Documents/cmps183/web2py/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/Users/rafihaque/Documents/cmps183/web2py/applications/hicappo/controllers/default.py", line 113, in <module>\n  File "/Users/rafihaque/Documents/cmps183/web2py/gluon/globals.py", line 419, in <lambda>\n    self._caller = lambda f: f()\n  File "/Users/rafihaque/Documents/cmps183/web2py/gluon/tools.py", line 3982, in f\n    return action(*a, **b)\n  File "/Users/rafihaque/Documents/cmps183/web2py/applications/hicappo/controllers/default.py", line 108, in payment_history\n    return dict(form=form)\nUnboundLocalError: local variable \'form\' referenced before assignment\n'
p11
s.